src/YQRichText.h

```diff
@@ -151,6 +151,12 @@ class YQRichText : public QFrame, public YRichText
      **/
     virtual void setHScrollValue( const std::string & newValue ) override;
 
+    /**
+     * Derived classes should implement this, method is used to trigger event
+     * like user has clicked link in the RichText
+     **/
+    virtual void clickLink( const std::string & url );
```

Comment:
This should become `activateLink()` in libyui, so this will need to be renamed here as well.

Reply:
renamed as in libyui.